### PR TITLE
docs: update "limit-conn" plugin docs 

### PR DESCRIPTION
**Pull request by zuiyangqingzhou**
### Description

Fixes https://github.com/apache/apisix/blob/master/docs/zh/latest/plugins/limit-conn.md#L32-L35

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by tzssangglass — DISMISSED** [review]

File: docs/zh/latest/plugins/limit-conn.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ title: limit-conn
...
-| default_conn_delay | number  | required |        | default_conn_delay > 0                                                                    | 默认的典型连接（或请求）的处理延迟时间。                                                                                                                                                                                     |
-| only_use_default_delay  | boolean | optional | false  | [true,false]                                                                              | 延迟时间的严格模式。 如果设置为`true`的话，将会严格按照设置的时间来进行延迟                                                                                                                                                                |
+| conn               | integer | 必须 |        | conn > 0                                                                                  | 允许的最大并发请求数。超过 `conn` 的限制、但是低于 `conn` + `burst` 的请求，将被延迟处理。                                                                                                                                               |
```
Comment (hf400159) [inline docs/zh/latest/plugins/limit-conn.md:32]:
```suggestion
| conn               | integer | 是  |        | conn > 0                                                                                  | 允许的最大并发请求数。超过 `conn` 的限制、但是低于 `conn` + `burst` 的请求，将被延迟处理。                                                                                                                                               |
```

File: docs/zh/latest/plugins/limit-conn.md [hunk 2]
```diff
@@ -29,10 +29,10 @@ title: limit-conn
...
+| burst              | integer | 必须 |        | burst >= 0                                                                                | 允许被延迟处理的并发请求数。                                                                                                                                                                                           |
+| default_conn_delay | number  | 必须 |        | default_conn_delay > 0                                                                    | 默认的典型连接（或请求）的处理延迟时间。                                                                                                                                                                                     |
+| only_use_default_delay  | boolean | 可选 | false  | [true,false]                                                                              | 延迟时间的严格模式。 如果设置为`true`的话，将会严格按照设置的时间来进行延迟                                                                                                                                                                |
```
Comment (hf400159) [inline docs/zh/latest/plugins/limit-conn.md:35]:
```suggestion
| only_use_default_delay  | boolean |  否   | false  | [true,false]                                                                              | 延迟时间的严格模式。 如果设置为`true`的话，将会严格按照设置的时间来进行延迟                                                                                                                                                                |
```

**Review by hf400159 — COMMENTED** [review]
Please change `required` to `是` and `optional` to`否`. Please modify all 'required' columns in this article.

File: docs/zh/latest/plugins/limit-conn.md (hunk 2 above)
Comment (zuiyangqingzhou) [reply docs/zh/latest/plugins/limit-conn.md]: > Please change `required` to `是` and `optional` to`否`. Please modify all 'required' columns in this article.

done

**Review by tzssangglass — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**